test/functional/wallet_conflicts.py

```diff
@@ -104,7 +105,7 @@ def test_block_conflicts(self):
 
         self.log.info("Verify, after the reorg, that Tx_A was accepted, and tx_AB and its Child_Tx are conflicting now")
         # Tx A was accepted, Tx AB was not.
-        assert conflicted_AB_tx["confirmations"] < 0
+        assert_greater_than(0, conflicted_AB_tx["confirmations"])
         assert conflicted_A_tx["confirmations"] > 0
```

Comment:
this looks like an actual assert_greater_than
```suggestion
        assert_greater_than(conflicted_A_tx["confirmations"], 0)
```